hack/verify-misspell.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+cd $ROOT
+
+source "${ROOT}/hack/lib.sh"
+
+hack::install_misspell
+
+ret=0
+find . -mindepth 1 -maxdepth 1 | grep -v -P '/(vendor)$' | xargs ${MISSPELL_BIN} -error || ret=$?
+if [ $ret -eq 0 ]; then
+    echo "Spellings all good!"
+else
+    echo "Found some typos, please fix thems!"
```

Comment:
Can this print out the misspelled words with the file and line numbers, like how k/k does it?